### PR TITLE
Test Group Mass Migration

### PR DESCRIPTION
**Pull request by chases2**
The result of the script at #13436 
Also includes manual fixups, in a separate commit
Original issue #13057

### CODE REVIEW

**Comment by chases2** [comment]
This change deletes a TODO on line 402 of config.yaml; I don't think those jobs have been automated, but I want to confirm with you

/cc @Katharine

**Comment by codenrhoden** [comment]
@chases2 Nice! I really only looked at the vSphere related stuff, but this saves me a lot of work I was going to do manually anyways. 👍

**Comment by Katharine** [comment]
Two major comments:

- You seem to have not migrated presubmits - why?
- There are now more jobs that have annotations visually attached to the wrong block with unwanted whitespace

**Comment by spiffxp** [comment]
>There are now more jobs that have annotations visually attached to the wrong block with unwanted whitespace

ref: https://github.com/kubernetes/test-infra/issues/13413

**Comment by chases2** [comment]
I've updated the script, but probably won't get to updating this result until Monday.
/hold

**Comment by chases2** [comment]
The script has been rerun with the changes made last Friday, and the manual fixup has been expanded to include unreasonable carriage-returns as well as misplaced comments

/hold cancel

File: config/jobs/kubernetes/sig-storage/sig-storage-gce-config.yaml [hunk 1]
```diff
@@ -223,6 +231,8 @@ presubmits:
           requests:
             memory: "6Gi"
```
Comment (Katharine) [inline config/jobs/kubernetes/sig-storage/sig-storage-gce-config.yaml]: Newline in the wrong place.

**Pull request merged**